### PR TITLE
org.apache.commons.commons-logging as dep in SDK feature

### LINKED ISSUE #3192: Bundle org.apache.commons.commons-logging cannot be resolved
Running Eclipse 2024-09 (4.33.0)
Eclipse Xtext 2.36.0.v20240825-0714 org.eclipse.xtext.sdk
on Linux Mint 21.3 Cinnamon

Just following the 15 mins tutorial (domainmodel) gives this error:
Bundle org.apache.commons.commons-logging cannot be resolved

### PR DESCRIPTION
**Pull request by LorenzoBettini**
Closes #3192

### CODE REVIEW

**Comment by cdietrich** [comment]
can you open a mwe pr

**Comment by iloveeclipse** [comment]
> can your team also evaluate if this will breaks something on your side
> this the past we were reducing the number of imports

Thanks for asking, not sure what exactly do you mean. 

We don't have `org.apache.commons.commons-logging` as dependency yet, neither in the Eclipse SDK nor in our product, is this known to be disruptive, or why do you ask?

**Comment by cdietrich** [comment]
https://ci.eclipse.org/xtext/job/xtext/job/lb_3192/3/display/redirect
once this has build. and you throw the zip into your
magic install scripts. will it work

**Comment by iloveeclipse** [comment]
> https://ci.eclipse.org/xtext/job/xtext/job/lb_3192/3/display/redirect once this has build. and you throw the zip into your magic install scripts. will it work

We will see it next week, I build now on Mondays :-)

File: org.eclipse.xtext.sdk.feature/feature.xml [hunk 1]
```diff
@@ -72,6 +72,8 @@ SPDX-License-Identifier: EPL-2.0
       <import plugin="org.eclipse.emf.ecore.source" version="2.26.0" match="greaterOrEqual"/>
       <import plugin="org.eclipse.emf.common.source" version="2.24.0" match="greaterOrEqual"/>
       <import feature="org.eclipse.emf.mwe2.language.sdk" version="2.19.0" match="greaterOrEqual"/>
+      <import plugin="org.apache.commons.commons-logging" version="1.3.4" match="greaterOrEqual"/>
```
Comment (szarnekow) [inline org.eclipse.xtext.sdk.feature/feature.xml:75]: Should the required bundles be picked up from the manifests?

Comment (LorenzoBettini) [reply org.eclipse.xtext.sdk.feature/feature.xml:75]: Currently none of our bundles require that.

**Comment by LorenzoBettini** [comment]
@cdietrich can I merge?

**Comment by cdietrich** [comment]
I can’t judge. Just merge

**Comment by iloveeclipse** [comment]
> > https://ci.eclipse.org/xtext/job/xtext/job/lb_3192/3/display/redirect once this has build. and you throw the zip into your magic install scripts. will it work
> 
> We will see it next week, I build now on Mondays :-)

Our build **would** work if **Xpect** would be able to build after this PR :-)

Currently it fails due:
```
[ERROR] Cannot resolve dependencies of project org.eclipse.xpect:org.eclipse.xpect:eclipse-plugin:0.3.0-SNAPSHOT
[ERROR]  with context {osgi.os=macosx, osgi.ws=cocoa, org.eclipse.update.install.features=true, osgi.arch=x86_64, org.eclipse.update.install.sources=true}
[ERROR]   You requested to install 'org.eclipse.equinox.p2.iu; org.apache.commons.logging 0.0.0, max=2147483647' but it could not be found: See log for details
[ERROR] -> [Help 1]
```

@mehmet-karaman : could you please update Xpect target file to make sure `org.apache.commons.logging` can be reolved?

**Comment by cdietrich** [comment]
See https://github.com/eclipse/Xpect/issues/361

**Pull request merged**